### PR TITLE
Update for extensions v2

### LINKED ISSUE #1: Update to extensions V2
Now that Chassis has support for Version 2 of extensions we can make use of the dependencies parameter.

### PR DESCRIPTION
**Pull request by BronsonQuick**
Fixes #1

### CODE REVIEW

File: chassis.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Extension version.
+#
+# All modern extensions should use 2 here. Version 1 uses a chassis.pp
+# instead, which requires a deprecated feature of Puppet.
+#
+# Values: 2
+version: 2
+
+dependencies:
+  - query-monitor
```
Comment (ntwb) [inline chassis.yaml:10]: This should be `querymonitor`, i.e. no `-`

https://github.com/Chassis/Query-Monitor/blob/master/chassis.pp#L1

**Review by ntwb — CHANGES_REQUESTED** [review]
See comment

File: chassis.yaml (hunk 1 above)
Comment (BronsonQuick) [reply chassis.yaml:10]: Negative Sir! If I change that we get: 
query

**Review by ntwb — APPROVED** [review]

**Comment by ntwb** [comment]
Thanks 👍

**Pull request merged**